common.sh

```diff
@@ -114,6 +114,11 @@ INT_IF=${INT_IF:-}
 #Root disk to deploy coreOS - use /dev/sda on BM
 ROOT_DISK_NAME=${ROOT_DISK_NAME-"/dev/sda"}
 
+# Provisioning network information
+PROVISIONING_NETWORK=${PROVISIONING_NETWORK:-172.22.0.0/24}
+PROVISIONING_HOST_IP=${PROVISIONING_HOST_IP:-172.22.0.1}
+PROVISIONING_NETMASK=${PROVISIONING_NETMASK:-255.255.255.0}
```

Comment:
Since we're specifying the CIDR for PROVISIONING_NETWORK, the netmask is probably redundant?

We can calculate it when needed e.g `$(ipcalc --netmask $PROVISIONING_NETWORK)`

Reply:
Thanks, good tip about ipcalc.  The only env var required now is $PROVISIONING_NETWORK.